src/vm/eventpipeblock.h

```diff
@@ -51,14 +51,13 @@ class EventPipeBlock final : public FastSerializableObject
         if (eventsSize == 0)
             return;
 
-        size_t currentPosition = pSerializer->GetCurrentPosition();
-        if (currentPosition % ALIGNMENT_SIZE != 0)
+        size_t requiredPadding = pSerializer->GetRequiredPadding();
+        if (requiredPadding != 0)
         {
             BYTE maxPadding[ALIGNMENT_SIZE - 1] = {}; // it's longest possible padding, we are going to use only part of it
-            unsigned int paddingLength = ALIGNMENT_SIZE - (currentPosition % ALIGNMENT_SIZE);
-            pSerializer->WriteBuffer(maxPadding, paddingLength); // we write zeros here, the reader is going to always read from the first aligned address of the serialized content
+            pSerializer->WriteBuffer(maxPadding, requiredPadding); // we write zeros here, the reader is going to always read from the first aligned address of the serialized content
```

Comment:
Missing cast from `size_t` to `unsigned int`? Or maybe make requiredPadding an `unsigned int` since it no longer has to keep track of huge numbers.

Reply:
This is what's failing the CI right now.

Reply:
Thanks for the catch